lib/shoryuken/options.rb

```diff
@@ -56,7 +58,12 @@ def ungrouped_queues
     end
 
     def polling_strategy(group)
-      strategy = (group == 'default' ? options : options[:groups].to_h[group]).to_h[:polling_strategy]
+      if options[:groups].nil?
+        strategy = (group == 'default' ? options : groups.to_h[group]).to_h[:polling_strategy]
+      else
+        strategy = (group == 'default' ? options : options[:groups].to_h[group]).to_h[:polling_strategy]
+      end
```

Comment:
hi @hkrdrm, thanks for the contribution. That's very welcome. 

WDYT about having an `if` for `default`? 

```ruby
strategy = \
  if group == 'default'
    options.to_h[:polling_strategy]
  elsif options[:groups].nil?
    # uses custom polling_strategy for the given group
    groups.to_h[group].to_h[:polling_strategy]
  else
    # uses default strategy
    options[:groups].to_h[group].to_h[:polling_strategy]
  end
```

Can you check if the comment makes sense? Late hours, I'm not sure if I got them right.

Reply:
Yeah that simplifies things and looks good to me.  On the comments for the else block I'd probably say its checking the option hash for polling strategy since its not necessarily default just whatever is coming in on the options hash.  I'll amend the commit and push it up.